src/amo/components/InstallButtonWrapper/index.js

```diff
@@ -118,6 +110,11 @@ export const InstallButtonWrapperBase = (props: InternalProps): React.Node => {
     );
   };
 
+  // This can be used to pass experiment-related query params to the
+  // GetFirefoxButton. Even though we do not have an active experiment,
+  // it is expected we will continue to experiment with this experience,
+  // so this logic remains in place.
```

Comment:
I don't think this is needed. The experiment HOC is removed so other code related to it should be removed as well, especially given that `experimentId` won't exist anymore (the HOC injects it but without HOC... nothing will be injected).

We can use git(hub) history if we ever need to look at the previous code.

Reply:
As we know for a fact that we are about to run another experiment (within a few weeks), I figured it was worth keeping some of this in place to avoid the cycle of "remove code, test, code review, add code back in, test, code review", which will take up time and effort, but I agree it's not ideal to have dead code in a component. I will take a different approach which will be to create a patch locally for the next experiment based on the current master branch, so at least the "add code back in" part will take little effort.